CMakeLists.txt

```diff
@@ -397,25 +406,22 @@ include_directories(
     #OpenDungeons includes
     ${CMAKE_SOURCE_DIR}/source
 
-    #General external includes
-    ${DEPENDENCIES_DIR}
-
     #Angelscript includes
-    ${ANGELSCRIPT_DIR}/angelscript/include
-    ${ANGELSCRIPT_SRC_DIR}
+    SYSTEM ${ANGELSCRIPT_DIR}/angelscript/include
```

Comment:
Ah ok, just found it:
http://www.cmake.org/cmake/help/v3.0/command/include_directories.html